diff --git a/dist/shared/unbuild.Cs2j_FHl.mjs b/dist/shared/unbuild.Cs2j_FHl.mjs
index 6da397241a3f2ca90d72d2dd98985690061cf172..2a5b8c3f22826e974d4c9af05b58a631ada2efa7 100644
--- a/dist/shared/unbuild.Cs2j_FHl.mjs
+++ b/dist/shared/unbuild.Cs2j_FHl.mjs
@@ -647,7 +647,7 @@ async function rollupBuild(ctx) {
           `const _jiti = jiti(null, ${serializedJitiOptions})`,
           "",
           `/** @type {import(${JSON.stringify(resolvedEntryForTypeImport)})} */`,
-          `const _module = await _jiti.import(${JSON.stringify(
+          `const _module = _jiti(${JSON.stringify(
             resolvedEntry
           )});`,
           hasDefaultExport ? "\nexport default _module;" : "",
